diff -ru AntTweakBar.orig/src/Makefile AntTweakBar/src/Makefile
--- AntTweakBar.orig/src/Makefile	2014-06-29 11:44:07.149872539 +0300
+++ AntTweakBar/src/Makefile	2014-06-29 12:08:35.597411653 +0300
@@ -15,13 +15,12 @@
 #OUT_DIR	= ../lib/debug
 
 
-CXX      	= gcc
 CXXFLAGS 	= $(CXXCFG) -Wall -fPIC -fno-strict-aliasing -D_UNIX -D__PLACEMENT_NEW_INLINE
 INCPATH  	= -I../include -I/usr/local/include -I/usr/X11R6/include -I/usr/include
-LINK     	= gcc
+LINK     	= $(CXX)
 #LIBS     	= -L/usr/X11R6/lib -L. -lglfw -lGL -lGLU -lX11 -lXxf86vm -lXext -lpthread -lm
 #LIBS     	= -L/usr/X11R6/lib -lGL -lX11 -lXxf86vm -lXext -lpthread -lm
-LIBS 		=
+LIBS 		= `pkg-config --libs glu` `pkg-config --libs gl` `pkg-config --libs sdl`
 AR       	= ar cqs
 RANLIB   	=
 TAR      	= tar -cf
@@ -40,7 +39,7 @@
 
 
 # name of the application:
-TARGET 		= AntTweakBar
+TARGET 		= anttweakbar
 
 # source files without extension:
 SRC_FILES	= TwColors.cpp TwFonts.cpp TwOpenGL.cpp TwOpenGLCore.cpp TwBar.cpp TwMgr.cpp TwPrecomp.cpp LoadOGL.cpp LoadOGLCore.cpp TwEventGLFW.c TwEventGLUT.c TwEventSDL.c TwEventSDL12.c TwEventSDL13.c TwEventSFML.cpp TwEventX11.c
@@ -68,7 +67,7 @@
 	@echo "===== Link $@ ====="
 	$(LINK) $(LFLAGS) -shared -Wl,-soname,lib$(TARGET)$(SO_EXT).$(SO_VERSION) -o $(OUT_DIR)/lib$(TARGET)$(SO_EXT) $(OBJS) $(LIBS)
 	$(SYMLINK) $(OUT_DIR)/lib$(TARGET)$(SO_EXT) $(OUT_DIR)/lib$(TARGET)$(SO_EXT).$(SO_VERSION)
-	$(AR) $(OUT_DIR)/lib$(TARGET)$(AR_EXT) $(OBJS) $(LIBS)
+	$(AR) $(OUT_DIR)/lib$(TARGET)$(AR_EXT) $(OBJS)
 
 .cpp.o:
 	@echo "===== Compile $< ====="
@@ -76,14 +75,13 @@
 
 .c.o:
 	@echo "===== Compile $< ====="
-	$(CXX) -c $(CXXFLAGS) $(INCPATH) -o $@ $<
+	$(CC) -c $(CXXFLAGS) $(INCPATH) -o $@ $<
 
 clean:
 	@echo "===== Clean ====="
 	-$(DEL_FILE) *.o
 	-$(DEL_FILE) *~ core *.core *.stackdump
 
-	
 ####### DEPENDENCIES
 
 TwColors.o:    TwPrecomp.h TwColors.h
